sql/7_0_2-to-7_0_3_upgrade.sql

```diff
@@ -113,3 +113,15 @@
 ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
 #EndIf
 
+#IfNotRow2D list_options list_id lists option_id default_open_tabs
+-- Create new list Default Open Tabs
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`) VALUES ('lists', 'Hide_Dashboard_Cards', 'Hide Dashboard Cards');
```

Comment:
above should be looking for option id `Hide_Dashboard_Cards`

Reply:
Is this what it should be
#IfNotRow lists option_id Hide_Dashboard_Cards